<launch>
	<node pkg="tanwaylidar_view" name="TanwayLidar_MainNode" type="TanwayLidar_MainNode" output = "screen">
		<param name="frame_id" type="string" value="TanwayTP" />
		<param name="topic" type="string" value="/tanwaylidar_pointcloud" />
		<param name="imu_topic" type="string" value="/tanwaylidar_imu" />
		<param name="LidarType" type="int" value="13"/> 

		<!--连接模式:仅支持"on-line"、"off-line"两种模式配置-->
		<param name="ConnectType" type="string" value="on-line" />

		<!--实时模式下配置-->
		<!--本机IP地址-->
		<param name="LocalHost" type="string" value="192.168.111.204" />

		<!--回放模式下配置-->
		<!--PCAP文件绝对路径-->
		<param name="PcapFilePath" type="string" value="/home/tanway/lidar_data/focus.pcap" />
		
	
		<!--雷达设备的IP地址-->
		<param name="LidarHost" type="string" value="192.168.111.51" />
		<!--雷达发送点云数据的目的端口（即设备接收点云数据的端口）-->
		<param name="LocalPointloudPort" type="int" value="5600" />
		<!--雷达发送DIF数据的目的端口（即设备接收DIF数据的端口）-->
		<param name="LocalDIFPort" type="int" value="5700" />

		<!--点云数据变换参数配置-->
		<param name="TransformRotateX" type="double" value="0.0" />	
		<param name="TransformRotateY" type="double" value="0.0" />	
		<param name="TransformRotateZ" type="double" value="0.0" />	
		<param name="TransformMoveX" type="double" value="0.0" />	
		<param name="TransformMoveY" type="double" value="0.0" />	
		<param name="TransformMoveZ" type="double" value="0.0" />	

		<!--组修正角度值1-->
		<param name="CorrectedAngle1" type="double" value="0.0" />		
		<!--组修正角度值2-->
		<param name="CorrectedAngle2" type="double" value="0.1" />	
		<!--组修正角度值3-->
		<param name="CorrectedAngle3" type="double" value="0.2" />	

		<!--是否开启拼接处理-->
		<param name="bJointabc" type="bool" value="false" />
		<!--分割节点1-->
		<param name="jointabc_node1" type="double" value="1.0" />
		<!--分割节点2-->
		<param name="jointabc_node2" type="double" value="10.0" />
		<!--单面区（近）A:0, B:1, C:2-->
		<param name="jointabc_one_face" type="int" value="0" />
		<!--双面区（远）A+B:1, A+C:2, B+C:3-->
		<param name="jointabc_two_face" type="int" value="3" />
	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find tanwaylidar_view)/rviz/tanwayrun.rviz" />
</launch>
